<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>711</width>
    <height>354</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>711</width>
    <height>354</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Vehicle</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/add.svg</normaloff>../icon/add.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="0" column="3">
    <widget class="QLabel" name="label_11">
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16pt;
	font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>Key Data Vehicle</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16pt;
	font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>Vehicle Data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Model Name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>86</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Make</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Chassis</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Region</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Transponder Type</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #CC0000;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color:#C0C0C0;
    color: white;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>PROX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NO PROX</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cloning</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="comboBox_4">
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Programming</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="comboBox_5">
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>AUSTRALIA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CANADA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EE.UU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EUROPA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ORIENTE MEDIO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AFRICA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_3">
     <item>
      <property name="text">
       <string>ALFA ROMEO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AUDI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BMW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CITROEN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DACIA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FIAT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HYUNDAI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KIA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NISSAN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OPEL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PEUGEOT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RENAULT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SEAT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOYOTA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VOLVO</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
